DALI_DEPS_VERSION

```diff
@@ -1 +1 @@
-459be38cfabd605a7199e2a6f346779b9eca181e
+be82b00b4b85e569865127b5c4c03eb740523f74
```

Comment:
You need to update that later anyway (sha will change after merge).

Reply:
Done